### DISCUSSION TITLE
Custom text new TSML UI

### DISCUSSION THREAD

**Discussion by bubfcg** [code4recovery/12-step-meeting-list#1423] [Q&A] [answered]
Just switched from the old UI to the new TSML UI. Looks so much better :)
But the widgets, like 'Meetings Top' and 'Meeting Detail Bottom' don't seem to work on the new UI. Is there any way to add custom text to the meeting list and meeting detail pages?

**Comment by joshreisner** [comment, marked as answer]
The `Meetings Top` and `Meetings Bottom` widgets should be working in TSML UI, but the `Meetings Detail Top` does not - this is because it is a single-page app
